### DISCUSSION TITLE
Failed at Prettier code formatter

### DISCUSSION THREAD

**Discussion by Asonjay** [alshedivat/al-folio#2178] [Q&A] [answered]
I tried to change the max-width of profile image so that my image will not cover the whole page while viewing from mobile device. I only modified this attribute in _base.scss:

``figure,
img {
  max-width: 55vw; (from 90vw)
}``

After committing, the workflow fails, this is the log:

> Run npx prettier . --check
Checking formatting...
[warn] _config.yml
[warn] _data/coauthors.yml
[warn] _layouts/about.liquid
[warn] _layouts/cv_embedded.liquid
[warn] _layouts/default.liquid
[warn] _layouts/redirect.liquid
[warn] _news/announcement_1.md
[warn] _pages/about_einstein.md
[warn] _pages/about.md
[warn] _pages/projects.md
[warn] _pages/publications.md
[warn] _projects/Chinese-Writing-Accessment.md
[warn] _projects/SalsaBot.md
[warn] _projects/Tele-Tube.md
[warn] _projects/TimeWise.md
[warn] _sass/_base.scss
[warn] _sass/_variables.scss
[warn] README.md
[warn] Code style issues found in 18 files. Run Prettier to fix.
Error: Process completed with exit code 1.

Can you help me fix this?

**Comment by george-gca** [comment, marked as answer]
Check information about Prettier in our [FAQ.md](https://github.com/alshedivat/al-folio/blob/master/FAQ.md).
